<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>serialAssistant</class>
 <widget class="QMainWindow" name="serialAssistant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>602</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>602</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>serialAssistant</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="1" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="serialDataTab">
       <attribute name="title">
        <string>Serial Data</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPlainTextEdit" name="recvEdit">
              <property name="minimumSize">
               <size>
                <width>700</width>
                <height>445</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLineEdit" name="beginMessageEdit">
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_7">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>Begin Message </string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="endMessageEdit">
                  <property name="maximumSize">
                   <size>
                    <width>120</width>
                    <height>25</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLabel" name="label_8">
                  <property name="minimumSize">
                   <size>
                    <width>100</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>70</width>
                    <height>25</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>End Message</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLineEdit" name="sendMessageEdit">
                <property name="minimumSize">
                 <size>
                  <width>350</width>
                  <height>50</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QPushButton" name="sendMessagePbt">
                  <property name="text">
                   <string>Send Message</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="clearMessagePbt">
                  <property name="text">
                   <string>Clear Message</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="maximumSize">
               <size>
                <width>162</width>
                <height>64</height>
               </size>
              </property>
              <property name="title">
               <string/>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>52</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="minimumSize">
                  <size>
                   <width>126</width>
                   <height>20</height>
                  </size>
                 </property>
                 <property name="font">
                  <font>
                   <family>DejaVu Sans</family>
                   <pointsize>11</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Serial Assisant</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>52</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout">
              <item row="0" column="0">
               <widget class="QComboBox" name="serialPortCb">
                <property name="maximumSize">
                 <size>
                  <width>86</width>
                  <height>25</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLabel" name="label">
                <property name="maximumSize">
                 <size>
                  <width>68</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>SerialPort</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QComboBox" name="baudRateCb">
                <property name="maximumSize">
                 <size>
                  <width>86</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>2</number>
                </property>
                <item>
                 <property name="text">
                  <string>4800</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>9600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>115200</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>68</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>BaudRate</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QComboBox" name="dataBitCb">
                <property name="maximumSize">
                 <size>
                  <width>86</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTipDuration">
                 <number>-10</number>
                </property>
                <property name="currentIndex">
                 <number>1</number>
                </property>
                <item>
                 <property name="text">
                  <string>7</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>8</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLabel" name="label_3">
                <property name="maximumSize">
                 <size>
                  <width>68</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>DataBits</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QComboBox" name="stopBitCB">
                <property name="maximumSize">
                 <size>
                  <width>86</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1.5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QLabel" name="label_4">
                <property name="maximumSize">
                 <size>
                  <width>68</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>StopBits</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QComboBox" name="ParityCb">
                <property name="maximumSize">
                 <size>
                  <width>86</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <item>
                 <property name="text">
                  <string>none</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QLabel" name="label_5">
                <property name="maximumSize">
                 <size>
                  <width>68</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Parity</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="searchSearchportPbt">
              <property name="text">
               <string>Search Serial Port</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="openSerialPbt">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Open Serial</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="closeSerialPbt">
                <property name="maximumSize">
                 <size>
                  <width>160</width>
                  <height>30</height>
                 </size>
                </property>
                <property name="text">
                 <string>Close Serial</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chartsTab">
       <attribute name="title">
        <string>Charts</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="0">
         <widget class="QChartView" name="chartview">
          <property name="minimumSize">
           <size>
            <width>630</width>
            <height>441</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>400</height>
             </size>
            </property>
            <property name="title">
             <string>GroupBox</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="runPbt">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Run</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopPbt">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Stop</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearMessageChartPbt">
              <property name="minimumSize">
               <size>
                <width>150</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>Clear Message</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
